pkg/services/auth/settings.go

```diff
@@ -0,0 +1,121 @@
+package auth
```

Comment:
This whole file is a bit redundant with `pkg/services/ldap/settings.go`, which is probably fine in the interest of minimizing changes in other packages.